### PR TITLE
IDE-716 "Collapse All" option for tree

### PR DESCRIPTION
**Pull request by dehilsterlexis**
fixed IDE-716

Signed-off-by: dehilsterlexis <david.dehilster@lexisnexis.com>

### CODE REVIEW

**Comment by dehilsterlexis** [comment]
@GordonSmith for review

File: wlib/RepositoryView.h [hunk 1]
```diff
@@ -479,6 +479,11 @@ class CRepositoryViewT
         m_Root->Expand();
         return true;
     }
+    void CollapseAll()
+    {
+        m_Root->Expand(TVE_COLLAPSE);
```
Comment (GordonSmith) [inline wlib/RepositoryView.h]: Should this be TVE_COLLAPSERESET?

Comment (dehilsterlexis) [reply wlib/RepositoryView.h]: You are correct. Added.

**Comment by dehilsterlexis** [comment]
@GordonSmith back to you.

**Pull request merged**